Mlem/Views/Shared/Posts/Expanded Post.swift

```diff
@@ -144,6 +144,8 @@ struct ExpandedPost: View {
                     showPostContext: false,
                     showCommentCreator: true
                 )
+                /// [2023.08] Manually set zIndex so child comments don't overlap parent comments on collapse/expand animations. `Int.max` doesn't work, which is why this is set to just some big value.
+                .zIndex(Double(99999 - comment.depth))
```

Comment:
Can you add this as an extension on double?

`Double.maxZIndex`

Reply:
Double has been extended =)